### PR TITLE
Update User Guide - List and Find

### LINKED ISSUE #53: Update User Guide: List all contacts, Find Contact By Name

### PR DESCRIPTION
**Pull request by marcusc55**
Update user guide for list and find commands.

Closes #53

### CODE REVIEW

**Review by FergusMok — APPROVED** [review]

File: docs/UserGuide.md [hunk 1]
```diff
@@ -87,11 +87,11 @@ Examples:
...
 * `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
 
-### Listing all persons : `list`
+### Listing all students: `list`
 
-Shows a list of all persons in the address book.
+Shows a list of all students matching the specified tags or list of all students if no tags are specified.
 
-Format: `list`
+Format: `list [TAG]`
```
Comment (caipng) [inline docs/UserGuide.md:94]:
```suggestion
Format: `list [TAG]...`
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -126,7 +126,6 @@ Format: `find KEYWORD [MORE_KEYWORDS]`
 Examples:
 * `find John` returns `john` and `John Doe`
 * `find alex david` returns `Alex Yeoh`, `David Li`<br>
-  ![result for 'find alex david'](images/findAlexDavidResult.png)
```
Comment (caipng) [inline docs/UserGuide.md:129]: We should probably remove that file if we are not going to be using it.

**Review by caipng — CHANGES_REQUESTED** [review]
2 minor issues, other than that LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (marcusc55) [reply docs/UserGuide.md]: Updated.

File: docs/UserGuide.md (hunk 2 above)
Comment (marcusc55) [reply docs/UserGuide.md:129]: File removed.

**Review by caipng — APPROVED** [review]
LGTM!

**Pull request merged**